build-logic/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     `kotlin-dsl`
+    `maven-publish`
```

Comment:
This shouldn't be here, maven publish is already present in the main builds, this one only applies to the build-logic.